.github/workflows/publish-docs.yaml

```diff
@@ -0,0 +1,24 @@
+name: Build and deploy documentation
+
+on:
+  workflow_dispatch:
+
+jobs:
+    publish-docs:
+      runs-on: ubuntu-latest
+      steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+            submodules: recursive
+      - name: .NET Setup
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 7.x
```

Comment:
I just meant "how come", as in "how come it would be nicer"?

Reply:
In my past experience working on other projects then using consistent versioning makes it easier to prioritise when to update.

Example net7 goes out of support before net6. If you know only net6 is used in our C# SDK then we can plan after this and update before it goes out of support. If instead we use different versions it’s almost impossible to know if you are up to date and often out of support catches you because you see issues in your pipelines.